### PR TITLE
Tutorial improvements

### PR DESCRIPTION
**Pull request by Treiblesschorle**

### CODE REVIEW

File: notebooks/1_-_Using_ImageJ/1_-_Fundamentals_of_ImageJ.ipynb [hunk 1]
```diff
@@ -164,7 +164,7 @@
     "\n",
     "<p><img src=\"ide-autocomplete.png\"></p>\n",
     "\n",
-    "Unfortunately, here in this notebook software, the autocompletion logic is not (yet!) advanced enough to fully offer such a feature—there is only rudimentary method completion support by pressing <kbd>tab</kbd>. Hence, the `NotebookService` also includes a handy method for inspecting an object's methods. Here is an example:"
+    "Unfortunately, here in this notebook software, the autocompletion logic is not (yet!) advanced enough to fully offer such a feature—there is only rudimentary method completion support by pressing <kbd>tab</kbd>. Hence, the `NotebookService` also includes a handy method for inspecting an object's methods. Here is an example which will show the vailable ethods of a List:"
```
Comment (imagejan) [inline notebooks/1_-_Using_ImageJ/1_-_Fundamentals_of_ImageJ.ipynb]: `s/ethods/methods/`

File: notebooks/1_-_Using_ImageJ/ImageJ_Ops/06_-_Evaluating_Expressions.ipynb [hunk 2]
```diff
@@ -105,7 +105,7 @@
    "source": [
     "You can also call any op in an `eval` statement as a function, using familiar function syntax.\n",
     "\n",
-    "Here is an example of the `eval` op being used to compute a [Difference of Gaussians](https://en.wikipedia.org/wiki/Difference_of_Gaussians):"
+    "The following is an example of the `eval` op being used to compute a [Difference of Gaussians](https://en.wikipedia.org/wiki/Difference_of_Gaussians). Inputs within the formula may be accessed via the `Map vars?` argument of the eval function. Thereby, the key of the map corresponds to the name of the variable that can be used in the formula."
```
Comment (imagejan) [inline notebooks/1_-_Using_ImageJ/ImageJ_Ops/06_-_Evaluating_Expressions.ipynb]: Wrong use of "thereby", no?

File: notebooks/1_-_Using_ImageJ/ImageJ_Ops/13_-_Frangi_Vesselness_Filter.ipynb [hunk 3]
```diff
@@ -77,15 +77,19 @@
     "\n",
     "//scales refers the the pixel distance at which the filter operates. Larger scales measure larger structures.\n",
     "//spacing refers to the physical distance between pixels. The default setting is {1, 1, 1...}\n",
-    "#@String(label=\"spacing: e.g. \\\"1, 1, 1\\\"\",value=\"1, 1, 1\") spacing\n",
+    "spacings = [1.0, 1.0, 1.0]\n",
```
Comment (imagejan) [inline notebooks/1_-_Using_ImageJ/ImageJ_Ops/13_-_Frangi_Vesselness_Filter.ipynb]: Why did you remove the script parameters?

Comment (Treiblesschorle) [reply notebooks/1_-_Using_ImageJ/ImageJ_Ops/13_-_Frangi_Vesselness_Filter.ipynb]: It was not used in the example. For the sake of the example its easier to just use a list. Also, then one can remove the parsing of the input String into a list.

File: notebooks/1_-_Using_ImageJ/ImageJ_Ops/13_-_Frangi_Vesselness_Filter.ipynb [hunk 4]
```diff
@@ -77,15 +77,19 @@
...
     "actualOutput = ArrayImgs.doubles(dims)\n",
     "ij.op().run(DefaultFrangi.class, actualOutput, input, spacings, 1)\n",
-    "[\"Output: \", actualOutput]"
+    "\n",
+    "// In order to make the vessels easier visible, we will use morphological dilation as explained in the previous tutorial.\n",
+    "import net.imglib2.algorithm.neighborhood.HyperSphereShape\n",
+    "shape = new HyperSphereShape(1)\n",
+    "dilate = ij.op().morphology().dilate(actualOutput, shape)\n",
+    "\n",
+    "[[\"Output: \":actualOutput, \"Dilated: \":dilate]]"
```
Comment (gselzer) [inline notebooks/1_-_Using_ImageJ/ImageJ_Ops/13_-_Frangi_Vesselness_Filter.ipynb]: Do we really want to use an `Op` in a notebook dedicated to a different `Op`?

Honestly I am not sure that it is worth fixing this notebook. There already exists a [Frangi Vesselness](https://github.com/imagej/tutorials/blob/a64836d43bd14922c1cdd190da46141a087c5ad5/notebooks/1_-_Using_ImageJ/Ops/filter/frangiVesselness.ipynb) notebook in the new Ops scheme (one notebook for every `Op`), so this notebook will probably be deleted later on.
